src/QtiItems/QtiItemAssetCloudFrontReplacer.php

```diff
@@ -0,0 +1,156 @@
+<?php
```

Comment:
Some fixes are needed from PSR-12 validation of this file and then we're good for this PR I think :)
```
 134 | ERROR   | [x] Expected 1 space(s) after closing parenthesis;
     |         |     found 0
 135 | ERROR   | [x] Expected 1 space(s) after IF keyword; 0 found
 144 | ERROR   | [x] Line indented incorrectly; expected at least 8
     |         |     spaces, found 7
 160 | ERROR   | [x] Expected 1 newline at end of file; 0 found
```

Reply:
sorry for that... my PHPStorm ignore this file...